x-pack/plugins/watcher/public/sections/watch_list/components/watch_list.tsx

```diff
@@ -133,11 +133,9 @@ const WatchListUi = ({ intl }: { intl: InjectedIntl }) => {
       actions: [
         {
           render: (watch: any) => {
-            const disabled = watch.isSystemWatch;
             return (
               <EuiButtonEmpty
                 iconType="pencil"
-                disabled={disabled}
```

Comment:
Sorry, I should have clarified that I think you made the right decision originally to disable the Edit button. But because we can't control users who manually navigate to the `/edit` route, we needed to display the info callout.

Reply:
ah, oops. i misunderstood. fixed!